app/models/user.rb

```diff
@@ -31,6 +31,9 @@ class User < ActiveRecord::Base
   # Gem ldap_authenticatable expects User to respond_to? :login. For us that's #username.
   alias_attribute :login, :username
 
+  # TODO: Update downstreams to use `suspended_at`
```

Comment:
What is the reason we are leaving this as a todo rather than including that change in this PR?

Reply:
UIC uses `deactivated_at` in some of its synchronization jobs. We'll want to change that once this merges down, but this provides a way to not break in the meantime.